skyuxconfig.json

```diff
@@ -1,16 +1,18 @@
 {
   "mode": "easy",
   "compileMode": "aot",
-  "auth": true,
+  "auth": false,
   "plugins": [
     "@blackbaud/skyux-builder-plugin-stache",
-    "@blackbaud/skyux-builder-plugin-auth-email-whitelist"
   ],
   "app": {
     "title": "Stache 2 Template"
   },
   "appSettings": {
     "stache": {
+      "editButton": {
+        "url": "{your-repo-url}"
```

Comment:
I think this is going to break having the curly brackets.  Likely why your tests failed.  I would just make this `"url": "your-repo-url"` instead.